File: Array/26 Remove Duplicates from Sorted Array  (not done)/removedup.js
var removeDuplicates = function(nums) {
    dupCount = 0;
    for (let i = 0; i < nums.length-1; i++) {
        // for (let j = i+1; j < nums.length; j++) {
        //     if (nums[i] == nums[j]) {
        //         dupCount++;
        //         nums.splice(nums[i], 1);
        //     }
        // }
        if (nums[i] == nums[i+1]) {
            dupCount++
            //nums.splice(nums[i], 1);
            nums.push(nums[i]);
        }
    }
    return nums
};


array1 = [1, 1, 2];
array2 = [0,0,1,1,1,2,2,3,3,4];
console.log(removeDuplicates(array1));
console.log(removeDuplicates(array2));

File: All Question overview/1 ~ 100/26 Remove Duplicates from Sorted Array/071318/removedup.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        dupCount = 0
        for i in range((len(nums)-1)):
          for j in range(i+1, len(nums)):
              if nums[i] == nums[0]:
                  dupCount += 1
                  del nums[j]
        return len(nums)


sl = Solution()
list1 = [1, 1, 2]
list2 = [0, 0, 1, 1, 0, 3, 0, 12]
print(sl.removeDuplicates(list1))
print(sl.removeDuplicates(list2))
